src/containers/Admin/Artists/index.js

```diff
@@ -7,92 +7,74 @@ import Header from '../../../components/Header';
-        fetchAllArtists().then(({data}) => {
-            this.setState({ artists: data })
-        });
-    }
+  useEffect(() => {
+    fetchAllArtists().then(({ data }) => {
+      setArtists(data);
+    });
+  });
 
-    toggleModal = () => {
-      this.setState(prevState => ({
-        show: !prevState.show
-      }))
-    }
+  const toggleModal = () => setShow(!show);
 
-    createArtist = () => {
-      const { artistName, artistUrl } = this.state;
-      createArtist(artistName, artistUrl);
-      this.toggleModal();
-    }
+  const createNewArtist = () => {
+    createArtist(artistName, artistUrl);
+    toggleModal();
+  }
 
-    render() {
-        const { artists, show } = this.state;
-        return (
-            <Fragment>
-                <Header />
-                <Container>
-                    <Col>
-                    <Wrapper>
-                      <Button style={{ float: 'right', margin: 4 }} onClick={this.toggleModal}>Criar</Button>
-                      <Table striped bordered hover>
-                        <thead>
-                          <tr>
-                            <th>Id</th>
-                            <th>Nome</th>
-                          </tr>
-                        </thead>
-                        <tbody>
-                          {artists.map(artist => (<tr><td>{artist._id}</td><td>{artist.name}</td></tr>))}
-                        </tbody>
-                      </Table>
-                      </Wrapper>
-                    </Col>
-                </Container>
-                <Footer />
-                <Modal show={show} >
-                  <Modal.Header>
-                    <Modal.Title>Criar Artista</Modal.Title>
-                  </Modal.Header>
-                  <Modal.Body>
-                    <FormWrapper>
-                      <FormControl
-                        placeholder="Nome do Artista"
-                        aria-label="Artista"
-                        aria-describedby="basic-addon1"
-                        onChange={e => this.setState({ artistName: e.target.value })}
-                        style={{ marginBottom: 12 }}
-                      />
-                      <FormControl
-                        placeholder="Imagem do Artista"
-                        aria-label="Imagem"
-                        aria-describedby="basic-addon1"
-                        onChange={e => this.setState({ artistUrl: e.target.value })}
-                      />
-                    </FormWrapper>
-                  </Modal.Body>
-                  <Modal.Footer>
-                    <Button variant="secondary" onClick={this.toggleModal}>
-                      Cancelar
-                    </Button>
-                    <Button variant="primary" onClick={this.createArtist}>
-                      Criar
-                    </Button>
-                  </Modal.Footer>
-                </Modal>
-            </Fragment>
-        );
-    }
-}
+  return (
+    <Fragment>
```

Comment:
Eu to assinalando coisas que eu mudaria também, então neste caso, o uso do `<Fragment>` pode ser utilizado na shorthand de `<>` e fechado com `</>`.

ps: lembra de retirar o import lá em cima do documento.